Tests/Tactics/CSE.lean

```diff
@@ -1,52 +1,165 @@
-import Tactics.CSE
+/-
+Copyright (c) 2024 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Author(s): Siddharth Bhat
 
+This file contains tests for the common subexpression elimination pass.
+-/
 import Arm.Insts.DPSFP.Insts
 import Specs.SHA512
-import Std.Tactic.BVDecide
 import Tactics.CSE
 
 
 /- ### Test that CSE succeeds and counts subexpressions correctly.
 
-We want to check that it correctly ees that there are:
+We want to check that it correctly sees that there are:
 - `16` occurrences of `y`,
 - `8` occurrences of `y + y`,
 - `4` occurrences of `(y + y) + (y + y)`.
 
 -/
 
-set_option trace.Tactic.cse.generalize true in
 set_option trace.Tactic.cse.summary true in
```

Comment:
Does setting this option give a verbose `make` output?

Reply:
No, because it's guarded by a `#guard_msgs`.